app/models/user.rb

```diff
@@ -1,6 +1,11 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  has_many :initial_answers
+  has_many :weekly_answers
+  has_many :members
```

Comment:
why? why not has one member, because he already has many teams, you can only have on member_id per team